src/Surfnet/StepupMiddleware/CommandHandlingBundle/SensitiveData/RightToObtainDataInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData;
+
+interface RightToObtainDataInterface
```

Comment:
Please consider to get only the field names with user data of the events with this interface and to do the aggregation of the data in a separate class to prevent the code duplication of merging data  in all concrete event classes.

Reply:
Hi Bas, Thanks for the feedback. This is exactly what we'll do next. This was mainly an exercise for Vadim to get the whitelisted data right. The recurring filter logic will be offloaded to a helper class that is responsible for that task.

I'll be sure to include you on a successive PR where we implement this 'the right way'